Dockerfile

```diff
@@ -89,6 +89,8 @@ EXPOSE 7090/udp
 EXPOSE 8887/tcp
 # SMA Energy Manager
 EXPOSE 9522/udp
+# GoodWe Wifi Inverter
+EXPOSE 8899/udp
```

Comment:
Bitte in Reichenfolge- 2 Zeilen nach oben ;)

Reply:
ok erledigt.

Reply:
ok erledigt.